### ISSUE TITLE
Show some type of error if Agent or PDK is not found when starting

### ISSUE THREAD

**Issue by rogermartensson** [lingua-pupuli/puppet-vscode#413]
I thought that it was a good idea to try PDK instead of the Puppet Agent.

So I installed PDK and uninstalled my Agent and wondered why I couldn't get the extension to use it. Some googling told me that I needed to set the puppet.installType to PDK.

It would've be nice if VSCode told me that using a Popup or something.

**Comment by jpogran** [comment]
Thanks for the feedback @rogermartensson, what version of the extension are you running? If it's not at least 0.11.0, you're missing out on some new things, namely back in July in PR #302 we added a message that pops up if the Puppet agent or PDK cannot be found.

We also have some guidance in the [Configuration section](https://github.com/lingua-pupuli/puppet-vscode/blob/master/README.md#configuration) of the readme that talks about how to set which one the extension uses, and where to look for the install directory.

**Comment by rogermartensson** [comment]
Strange.. I'm at 0.13.0 and I didn't see the warning at all. How often do this warning show? Every time I enter a .pp file, at install or at startup? If it's at startup then I could understand why I may have missed it. I don't restart VSCode that often. :)

When I upgraded I had an older Agent running. Some time after the install I uninstalled the agent and installed the latest PDK since you added support for it and wanted to try it. I did restart VSCode after this and can't remember a warning. Doesn't mean it may not have been shown. Only that I might have missed seeing it.
